R/prep_net_zero_commitments.R

```diff
@@ -16,31 +16,43 @@
 prep_net_zero_commitments <- function(total_portfolio,
```

Comment:
We need to document the parameters that go into this function since it is a user-facing function.